src/pages/regulatory/index.js

```diff
@@ -206,92 +200,90 @@ const Regulatory = () => {
+                            </ResponsiveGrid>
+                            <CssGridColumn>
+                                <Desktop>
+                                    <StyledHeader as="h4" type="sub-section-title">
+                                        {localize('Deriv (Europe) Limited')}
+                                    </StyledHeader>
+                                </Desktop>
+                                <Text mt="0.8rem" mb="1.6rem" max_width="58.8rem">
+                                    <Localize
+                                        translate_text="Deriv (Europe) Limited, W Business Centre, Level 3, Triq Dun Karm, Birkirkara BKR 9033, Malta, is licensed for synthetic indices by the Malta Gaming Authority (<0>licence no. MGA/B2C/102/2000</0>), by the Gambling Commission for clients in Great Britain under (<1>account no. 39495</1>), and by the Revenue Commissioners for clients in Ireland (licence no. 1010285)."
+                                        components={[
+                                            <LinkText
+                                                key={0}
+                                                weight="bold"
+                                                target="_blank"
+                                                href="/regulatory/Deriv_(Europe)_Limited.pdf"
+                                                rel="noopener noreferrer"
+                                            />,
+                                            <ExternalBoldLink
+                                                external="true"
+                                                key={1}
+                                                weight="bold"
+                                                target="_blank"
+                                                to="https://beta.gamblingcommission.gov.uk/public-register/business/detail/39495"
+                                                rel="noopener noreferrer"
+                                            />,
+                                        ]}
+                                    />
+                                </Text>
+                                <Text max_width="58.8rem">
+                                    {localize(
+                                        'Clients in the European Union who wish to trade gambling products can have accounts under Deriv (Europe) Limited.',
+                                    )}
+                                </Text>
+                            </CssGridColumn>
+                            <ResponsiveGrid align="flex-start">
+                                <Mobile>
+                                    <StyledHeader as="h4" type="sub-section-title" mb="1.6rem">
+                                        {localize('Deriv (MX) Ltd')}
+                                    </StyledHeader>
+                                </Mobile>
+                                <Img src={IOM} alt="IOM" />
```

Comment:
Can we use an `alt` that is more descriptive 🙏🏻

Reply:
sure. the code was added before. done.